mobile/src/components/uam-groups-config/uam-groups-edit.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsUpdateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from '~/components/uam-groups-config/common/constants/constants';
+import {
+  PermissionsTable,
+  UsersTable,
+} from '~/components/uam-groups-config/components/components';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { groupsCreationActions, uamGroupEditActions } from '~/store/actions';
+import { getUsers } from '~/store/uam/actions';
+import { groupUpdateParams } from '~/validation-schemas/validation-schemas';
+
+import { styles } from './styles';
+
+const UAMGroupsEdit: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const { page: usersPage, handlePageChange: handleUserPageChange } =
+    usePagination();
+
+  const {
+    page: permissionsPage,
+    handlePageChange: handlePermissionsPageChange,
+  } = usePagination();
+
+  const { control, handleSubmit, errors } = useAppForm<GroupsUpdateRequestDto>({
+    defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+    validationSchema: groupUpdateParams,
+  });
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const { group } = useAppSelector((state) => state.uamGroupEdit);
+
+  const { items: permissionIds, handleToggle: handleTogglePermissions } =
+    useSelectedItems<number>(group?.permissionIds as number[]);
+
+  const { items: userIds, handleToggle: handleToggleUsers } =
+    useSelectedItems<number>(group?.userIds as number[]);
```

Comment:
Won't it be better to have nullish coalescing, if `group` is possibly `null`? (same about default values for all other props that could possibly be undefined under the `null`)
```suggestion
    useSelectedItems<number>(group?.permissionIds ?? []);

  const { items: userIds, handleToggle: handleToggleUsers } =
    useSelectedItems<number>(group?.userIds ?? []);
```